plugins/pencilblue/include/routes.js

```diff
@@ -60,6 +60,34 @@ module.exports = function Routes(pb){
         },
         {
             method: 'get',
+            path: '/admin/localization',
+            auth_required: true,
+            access_level: pb.SecurityService.ACCESS_EDITOR,
+            inactive_site_access: true,
+            controller: path.join(pb.config.docRoot, 'plugins', 'pencilblue', 'controllers', 'admin','sites','localizations.js'),
+            content_type: 'text/html'
+        },
+        {
+            method: 'post',
+            path: '/actions/admin/localization',
```

Comment:
We'd like this to be: ```/api/localizations/overrides``` to fall in line with the other localization API end-points